libdevice/sanitizer_utils.cpp

```diff
@@ -537,6 +588,23 @@ inline uptr __asan_region_is_poisoned(uptr beg, uint32_t as, size_t size) {
   return 0;
 }
 
+constexpr size_t AlignMask(size_t n) {
+  switch (n) {
+  case 1:
+    return 0;
+  case 2:
+    return 1;
+  case 4:
+    return (1 << 2) - 1;
+  case 8:
+    return (1 << 3) - 1;
+  case 16:
+    return (1 << 4) - 1;
```

Comment:
Simplify the code by returning n-1 for above cases?
Also, the `default` case will never run since n won't be other values, right?

Reply:
done